config/default/forms/app/pregnancy.xml

```diff
@@ -961,998 +842,940 @@
+      <bind nodeset="/data/inputs/contact/patient_id" type="string"/>
+      <bind nodeset="/data/inputs/contact/date_of_birth" type="string"/>
+      <bind nodeset="/data/inputs/contact/sex" type="string"/>
+      <bind nodeset="/data/inputs/contact/parent/_id" type="string"/>
+      <bind nodeset="/data/inputs/contact/parent/parent/contact/chw_name" type="string"/>
+      <bind nodeset="/data/inputs/contact/parent/parent/contact/phone" type="string"/>
+      <bind nodeset="/data/patient_age_in_years" type="string" calculate="floor( difference-in-months( ../inputs/contact/date_of_birth, today() ) div 12 )"/>
+      <bind nodeset="/data/patient_uuid" type="string" calculate="../inputs/contact/_id"/>
+      <bind nodeset="/data/patient_id" type="string" calculate="../inputs/contact/patient_id"/>
+      <bind nodeset="/data/patient_name" type="string" calculate="../inputs/contact/name"/>
+      <bind nodeset="/data/patient_short_name" type="string" calculate="coalesce(../inputs/contact/short_name, ../summary/custom_translations/custom_woman_label)"/>
+      <bind nodeset="/data/patient_short_name_start" type="string" calculate="coalesce(../inputs/contact/short_name, ../summary/custom_translations/custom_woman_start_label)"/>
+      <bind nodeset="/data/lmp_date_8601" type="string" calculate="../gestational_age/g_lmp_date_8601"/>
+      <bind nodeset="/data/edd_8601" type="string" calculate="../gestational_age/g_edd_8601"/>
+      <bind nodeset="/data/days_since_lmp" type="string" relevant="../lmp_date_8601 != ''" calculate="floor(decimal-date-time(today()) - decimal-date-time(../lmp_date_8601))"/>
+      <bind nodeset="/data/weeks_since_lmp" type="string" relevant="../lmp_date_8601 != ''" calculate="round(../days_since_lmp div 7, 2)"/>
+      <bind nodeset="/data/weeks_since_lmp_rounded" type="string" relevant="../lmp_date_8601 != ''" calculate="floor(../days_since_lmp div 7)"/>
+      <bind nodeset="/data/lmp_method_approx" type="string" calculate="if(selected(../gestational_age/register_method/lmp_method, 'method_approx'), 'yes', 'no')"/>
+      <bind nodeset="/data/hiv_status_known" type="string" calculate="../safe_pregnancy_practices/hiv_status/hiv_tested"/>
+      <bind nodeset="/data/deworming_med_received" type="string" calculate="../safe_pregnancy_practices/deworming/deworming_med"/>
+      <bind nodeset="/data/tt_received" type="string" calculate="../safe_pregnancy_practices/tetanus/tt_imm_received"/>
+      <bind nodeset="/data/t_pregnancy_follow_up_date" type="string" calculate="../anc_visits_hf/anc_visits_hf_next/anc_next_visit_date/appointment_date"/>
+      <bind nodeset="/data/t_pregnancy_follow_up" type="string" calculate="../anc_visits_hf/anc_visits_hf_next/anc_next_visit_date/appointment_date_known"/>
+      <bind nodeset="/data/t_danger_signs_referral_follow_up_date" type="string" calculate="format-date-time(decimal-date-time(today()) + 3, &quot;%Y-%m-%d&quot;)"/>
```

Comment:
This is the source of the change to the `pregnancy.xlsx` file.  I think somehow a previous update to that file did not get committed properly (or got lost in a merge conflict) because the original xml did not actually match the original xlsx for this `calculate`.  Basically the xml was correctly using the `format-date-time` function in this calculate (as you see here), but the xlsx instead had the `date-time(decimal-date-time(today()) + 3)` (which also includes the _time_). 

So, I have just updated the `pregnancy.xlsx` to match what was already in the xml.